packages/mermaid/src/diagrams/gantt/ganttRenderer.js

```diff
@@ -611,7 +642,7 @@ export const draw = function (text, id, version, diagObj) {
             topXAxis.ticks(timeDay.every(every));
             break;
           case 'week':
-            topXAxis.ticks(timeWeek.every(every));
+            topXAxis.ticks(timeMonday.every(every));
```

Comment:
shouldn't be similar as the original?

Reply:
`timeWeek` is an alias for `timeSunday` (see https://github.com/d3/d3-time#timeWeek). Feel like this is the most explicit, and it shouldn't affect functionality one bit.

Reply:
But you changed `timeWeek` to `timeMonday` and not `timeSunday`, so the behavior is different right ?

Reply:
Yes, I'm sorry, I missed the `topXAxis` part. I've added the same code as in `bottomXAxis`, which should have been there from the beginning. Thanks for catching this.